classes/class-wc-connect-taxjar-integration.php

```diff
@@ -1628,9 +1628,9 @@ private function check_for_incorrect_california_tax_nexus( $response_body, $cach
 			$log_suffix    = 'in cached response.';
 		}
 
-		$to_state   = $response_body->tax->jurisdictions->state;
-		$to_country = $response_body->tax->jurisdictions->country;
-		$has_nexus  = $response_body->tax->has_nexus;
+		$to_state   = isset( $response_body->tax->jurisdictions->state ) ? $response_body->tax->jurisdictions->state : '';
+		$to_country = isset( $response_body->tax->jurisdictions->country ) ? $response_body->tax->jurisdictions->country : '';
+		$has_nexus  = isset( $response_body->tax->has_nexus ) ? $response_body->tax->has_nexus : false;
```

Comment:
What do you think if we do this
```suggestion
		$to_state   = isset( $response_body->tax->jurisdictions->state ) ? $response_body->tax->jurisdictions->state : 'not set';
		$to_country = isset( $response_body->tax->jurisdictions->country ) ? $response_body->tax->jurisdictions->country : 'not set';
		$has_nexus  = isset( $response_body->tax->has_nexus ) ? $response_body->tax->has_nexus : null;
```

Reply:
yeah I think we can. And also we can elevate this further by adding `throw` if some of those are not set. Because this method is wrapped by `try..catch`. WDYT?

Reply:
@bartech , can you check the latest commit on this PR? I have added your suggestion code and add throw exception code.